.github/workflows/code-analysis.yml

```diff
@@ -0,0 +1,39 @@
+name: PHP Code Analysis For Altapay Wordpress Plugin
+
+on:
+  pull_request:
+    branches: [ main ]
+  push:
+    branches: [ main ]
+
+jobs:
+  php-stan:
+    runs-on: ubuntu-latest
+    steps:
+     
+     - name: Install wordpress
+       run: |
+         curl -O https://wordpress.org/latest.tar.gz
+         tar xzvf latest.tar.gz 
+         rm latest.tar.gz
+         pwd
+         ls -al
```

Comment:
Is this intentionally left in?

Reply:
it was useful initially to know contents of dir, but now it can be removed